### ISSUE TITLE
disjunctive_strict: predicate cumulative not found

### ISSUE THREAD

**Issue by oboudry** [MiniZinc/libminizinc#183]
When running existing code with MiniZinc 2.1.6, I'm getting the following error:

```
Compiling suitor_schedule.mzn with data suitor_scheduleA_0.dzn
C:\Program Files\MiniZinc IDE (bundled)/share/minizinc/gecode/disjunctive_strict.mzn:7:
MiniZinc: type error: no function or predicate with this signature found: `cumulative(array[int] of var int,array[int] of var int,array[int] of int,int)'
Finished in 124msec
```

If in my code, I replace: `include "disjunctive.mzn";` with `include "globals.mzn";` the problem is fixed.

As I'm not directly using any cumulative predicates directly in my code, I'm suspecting there may be a missing include in the disjunctive_strict.mzn file causing this error.

I did not paste the code as it's part of the Coursera assignments. Can supply it separately if needed.

**Comment by Dekker1** [comment]
Thank you for your bug report. We found the bug. It's a missing include in the Gecode globals library. As you mentioned using `include "globals.mzn"` will circumvent the problem as you will have all the necessary includes.

We'll make sure to include a fix to the library in the next release!

**Comment by guidotack** [comment]
This has been fixed in the latest release 2.1.7.
